pkg/kubelet/dockertools/docker.go

```diff
@@ -691,3 +693,63 @@ type ContainerCommandRunner interface {
 	RunInContainer(containerID string, cmd []string) ([]byte, error)
 	GetDockerServerVersion() ([]uint, error)
 }
+
+type DockerCache interface {
+	KubeletRunningDockerContainers() (DockerContainers, error)
+}
+
+func NewDockerCache(client DockerInterface) DockerCache {
```

Comment:
this should also return a possible error and the error below we can return if that happens.